src/main/java/seedu/address/model/person/Person.java

```diff
@@ -77,12 +77,23 @@ public boolean isSamePerson(Person otherPerson) {
     /**
      * Sets the person's archive status to true. It's equivalent to archive the person.
```

Comment:
Grammar: It is equivalent to *the person being archived) OR (having archived the person).

Reply:
Will change this part, thank you!